seatunnel-connectors-v2/connector-tablestore/src/main/java/org/apache/seatunnel/connectors/seatunnel/tablestore/sink/TablestoreSinkClient.java

```diff
@@ -120,7 +122,8 @@ synchronized void flush() throws IOException {
 
     private void checkFlushException() {
         if (flushException != null) {
-            throw new RuntimeException("Writing items to Tablestore failed.", flushException);
+            throw new TablestoreConnectorException(TablestoreConnectorErrorCode.FLUSH_DATA_FAILED,
```

Comment:
CommonErrorCode.FLUSH_DATA_FAILED is better, please remove this redundant error code.